src/main/java/seedu/address/model/room/UniqueRoomList.java

```diff
@@ -49,13 +49,13 @@ public void setRoom(Room room) {
                 internalList.add(roomToAdd);
                 rooms.add(room);
             }
-            rooms.add(room);
-            internalList.add(room);
+            //rooms.add(room);
```

Comment:
Not sure if you still need these 2 lines of code since you commented out, but if you don't need it anymore, don't forget to delete it!